Custom pickup requests

lauriek334 opened the pull request.

taleon reviewed (approved):
1) General note, Drupal PHP uses snake case for function and variable names, not camel case
2) There are a couple areas where arborcat_create_pickup_request_record() is called and the indentation is very high. Generally we have these one tab over like an array
3) There is debug in here still
4) I'm not sure what the service is providing that another module function couldn't have done. This does abstract it out, though the functionality isn't global enough to need a service since this is already unique to arborcat.

All seems good, though!

taleon commented:
The pickup appointment form submit which calls the custom appointment function from the arborcat module does not return an error if the catch block is hit.

lauriek334 commented:
Return error fixed if the catch block hit.
Removed the CustomPickupRequests service and replaced with direct arborcat.module method: arborcat_custom_pickup_request
Debug removed.
